projects/uitdatabank/models/event-name-put.json

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "object",
+  "title": "event.name.put",
+  "additionalProperties": false,
+  "minProperties": 1,
+  "maxProperties": 1,
```

Comment:
Why the `minProperties` and `maxProperties`? That seems only relevant if you use `additionalProperties: {...}` (so with a schema, not `true`/`false`, to enable a variable list of properties names)

Reply:
Indeed, good point!